autonat.go

```diff
@@ -426,6 +431,13 @@ func (as *AmbientAutoNAT) getPeerToProbe() peer.ID {
 	return candidates[0]
 }
 
+func (as *AmbientAutoNAT) Close() error {
```

Comment:
Should this method get added to `interface.go` so that callers, which will typically have an `AutoNAT` rather than the concrete implementation, will be able to call it?

Reply:
Yes, thank you for catching this!